lib/install/validate/network.go

```diff
@@ -32,86 +32,103 @@ import (
 	"golang.org/x/net/context"
 )
 
-func (v *Validator) network(ctx context.Context, input *data.Data, conf *config.VirtualContainerHostConfigSpec) {
+func (v *Validator) addNetworkHelper(ctx context.Context, conf *config.VirtualContainerHostConfigSpec, netName, epName, contNetName string, def bool) error {
```

Comment:
would prefer the old format actually, although with some duplication. This new format is difficult to understand. 😸